app/controllers/admin/basic_projects_controller.rb

```diff
@@ -24,7 +24,13 @@ def index
   def show
     @project = BasicProject.find(params[:id])
     authorize @project
-    prep_form_vars
+
+    case @tab = params[:tab] || "details"
+    when "details"
+      prep_form_vars
+    end
+
+    @tabs = ["details", "timeline_table", "timeline_list", "logs", "calendar"]
```

Comment:
Since the timeline list will be going away, I suggest we name the table one just "timeline". Then when "timeline_list" goes away, it won't look weird.

The actual label on the tab can stay as "Timeline - Table" for now.